src/display/text_layer.js

```diff
@@ -226,7 +528,63 @@ var renderTextLayer = (function renderTextLayerClosure() {
           self._renderTimer = null;
         }, timeout);
       }
-    }
+    },
+
+    expandTextDivs: function TextLayer_expandTextDivs(expandDivs) {
+      if (!this._enhanceTextSelection || !this._renderingDone) {
+        return;
+      }
+      if (!this._expanded) {
+        expand(this._bounds, this._viewport);
+        this._expanded = true;
+      }
+      this._bounds.length = 0;
```

Comment:
Please move this into the `if(!this._expanded) { ... }` block just above, since it's not necessary to do it more than once.